app/src/main/java/io/gnosis/safe/utils/TxUtils.kt

```diff
@@ -27,18 +27,18 @@ fun TransactionStatus.isCompleted(): Boolean =
         TransactionStatus.CANCELLED -> true
     }
 
-fun TransactionInfoViewData.formattedAmount(balanceFormatter: BalanceFormatter): String =
+fun TransactionInfoViewData.formattedAmount(chain: Chain, balanceFormatter: BalanceFormatter): String =
     when (val txInfo = this) {
         is TransactionInfoViewData.Custom -> {
-            balanceFormatter.formatAmount(txInfo.value, false, 18, BuildConfig.NATIVE_CURRENCY_SYMBOL)
+            balanceFormatter.formatAmount(txInfo.value, false, chain.currency.decimals, chain.currency.symbol)
         }
         is TransactionInfoViewData.Transfer -> {
             val incoming = txInfo.direction == TransactionDirection.INCOMING
             val decimals: Int = when (val transferInfo = txInfo.transferInfo) {
                 is TransferInfo.Erc20Transfer -> {
                     transferInfo.decimals ?: 0
                 }
-                is TransferInfo.EtherTransfer -> 18
+                is TransferInfo.NativeTransfer -> 18
```

Comment:
I may be misunderstanding the syntax, but shouldn't the native transfer decimals taken from the current network's native currency details?

Reply:
definitely, missed that one